interface/frontend/annotate_and_segment.html

```diff
@@ -0,0 +1,21 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block tabs %}
```

Comment:
Whilst it makes no difference, I always do

```
{% block whatever %}
{{ block.super }}
content
{% endblock %}
```

Feels ... "safer"...

Reply:
Thanks again for that very valuable input! Now, the whole `settings` module can be accessed from the templates and the correct `url` template tags are used.